Correct Mistake in MariaDB CLI Command

apeschel opened the pull request:
There cannot be a space between `-p` and the password. This PR removes the space to correct the command used to connect to the MariaDB database.

sameersbn commented:
/ok-to-test

sameersbn commented:
@apeschel Good catch. Could you please complete the CLA https://github.com/kubernetes/charts/pull/1543#issuecomment-317520163

apeschel committed:
Correct Mistake in MariaDB CLI Command

There cannot be a space between `-p` and the password. This PR removes the space to correct the command used to connect to the MariaDB database.

apeschel commented:
Hello @sameersbn,

A few things:

1. The original commit had wrong email, it has been corrected.
2. I tried to follow the CLA FAQ for signing up as an organization, but there are problems with the system which make it impossible.

To elaborate on the problem:

* I have an existing account on identity.linuxfoundation.org that is tied to my github account
* The CLA requires login to identity.linuxfoundation.org via github account
* My github account is set up with two emails -- personal and organization
* My identity.linuxfoundation.org email is set to my personal email
* The identity.linuxfoundation.org account only allows a single email to be registered
* I would prefer not to change my email on my identity.linuxfoundation.org account to my organization email, since it was originally a personal account.

The CLA system provides no method for me to register a CLA with my organization email used in this commit. The FAQ recommends posting in this thread: https://github.com/kubernetes/kubernetes/issues/27796
However, there is someone in there who posted the same problem 7 months ago, and it has not even been acknowledged.

In short, the CLA system in use here is kind of broken at the moment.

apeschel commented:
I've filed a ticket with helpdesk@rt.linuxfoundation.org  as well.

unguiculus commented:
Marking this as stale. Please update PR within one week.

unguiculus commented:
What's the status with your CLA?

apeschel commented:
@unguiculus I've signed the CLA, but it looks like this PR is no longer needed.
